src/SonarScanner.MSBuild.PreProcessor/Protobuf/AnalysisCacheMsg.proto

```diff
@@ -0,0 +1,13 @@
+﻿/*
+  In order to update the generated file run
+  .\protoc.exe --proto_path=[sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf --csharp_out=[sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf [sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf\AnalysisCacheMsg.proto
+ */
+syntax = "proto3";
+
+option java_package = "org.sonar.scanner.protocol.internal";
+option optimize_for = SPEED;
+option csharp_namespace = "SonarScanner.MSBuild.PreProcessor.Protobuf";
+    
```

Comment:
Let's name this file just `AnalysisCache.proto`, as there might be more structures inside in the future. Or maybe just `Cache.proto` ?